resources/v9/01_books_languages.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE books_languages (
+  id SERIAL PRIMARY KEY,
+  language_id integer NOT NULL,
+  book_id integer NOT NULL,
+  created_at timestamp 
+    WITH time zone DEFAULT NOW
+    (),
+    updated_at timestamp
```

Comment:
we need a trigger here to update this right?

Reply:
I already added that in the next PR

Reply:
the PR #10